linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/rule.yml

```
@@ -39,11 +39,13 @@ identifiers:
     cce@rhel8: CCE-89455-0
     cce@sle12: CCE-83207-1
     cce@sle15: CCE-85591-6
+    cce@rhel7: CCE-85880-3
```

```suggestion
    cce@rhel7: CCE-85880-3
    cce@rhel8: CCE-89455-0
    cce@sle12: CCE-83207-1
    cce@sle15: CCE-85591-6
```

We have a test that checks if identifiers are sorted. The suggestion only moves `cce@rhel7` to the top of the list.